recipes/haskell-tng

```diff
@@ -0,0 +1,5 @@
+(haskell-tng
+ :repo "tseenshe/haskell-tng.el"
+ :branch "tng"
+ :fetcher gitlab
+ :files (:defaults "snippets"))
```

Comment:
Is `haskell-tng-testutils.el` only used for testing?  If so, you may need to explicitly :exclude it in this recipe.

Reply:
Disregard -- I missed that it was in a subdirectory. :)